projects/BFDR/BirthRecord_fieldsAndCreateMethods.cs

```diff
@@ -138,6 +138,9 @@ private void CreateAttendant()
             Attendant.Meta.Profile = attendant_profile;
             // Not linked to Composition or inserted in bundle, since this is run before the composition exists.
         }
+
+        /// <summary>Emerging Issues.</summary>
+        protected Observation EmergingIssues;
```

Comment:
the Natality-feature branch contains some helper functions for observations. If you use the helpers, you don't need to define this observation field here. Instead, the helpers will dynamically create and parse the observation